### DISCUSSION TITLE
[Accordion] How to make animated accordion responsive ?

### DISCUSSION THREAD

**Discussion by oggnimodd** [radix-ui/primitives#1311] [Help] [answered]
Hi guys!

I am following the [docs](https://www.radix-ui.com/docs/primitives/components/accordion#animating-content-size) to create an animated accordion. The animation works, but when resizing the window, the opened AccordionContent seems to have a fixed height, and is not responsive. AccordionContent will readjust its height only when accordion is reopened again

https://user-images.githubusercontent.com/64012087/161979974-444101e8-8416-4130-a439-83d3c992e82c.mp4

here's the [sandbox](https://codesandbox.io/s/busy-carlos-6kbu8z?file=/src/App.js)

**Comment by andy-hook** [comment, marked as answer]
Hi @dominggo1999 , this appears to be caused by the use of `animation-fill-mode: forwards` which persists the height post animation. We started recommending this as a way to [resolve flicker in React 18](https://github.com/radix-ui/primitives/pull/1125), though since you’ve brought this is our attention I’m wondering if it remains the best solution.

  In the meantime a simple workaround could be to remove `forwards` from the `open` animation as that direction doesn't appear to particularly suffer from the issue.
  
  https://codesandbox.io/s/tender-bash-fflezd

**Reply by oggnimodd** [reply]
@andy-hook yeah, using `animation-fill-mode: forwards` feels like a hack, but it gets the work done for now. Thank you for answering
